<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Feeds</class>
 <widget class="QMainWindow" name="Feeds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>obs - Select Feeds</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="all_feeds_radio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All feeds</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="update_Feeds_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Feeds</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="custom_feeds_radio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom feeds</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="packages_cb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Packages</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="luci_cb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Luci</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="routing_cb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Routing</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="telephony_cb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Telephony</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="management_cb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Management</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="targets_cb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Targets</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="install_Feeds_Button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Install Feeds</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next_b">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
